conceptnet5/vectors/evaluation/compare.py

```diff
@@ -20,18 +22,18 @@ def load_any_embeddings(filename):
 
 
 def compare_embeddings(filenames, subset='dev', tune_analogies=True):
-    embeddings = [
-        load_any_embeddings(filename) for filename in filenames
-    ]
     results = []
-    for frame in embeddings:
-        wordsim_results = wordsim.evaluate(frame, subset=subset)
-        analogy_results = analogy.evaluate(frame, ANALOGY_FILENAME, tune_analogies=True)
+    for filename in filenames:
+        print(filename)
```

Comment:
This step generally takes a long time and I wanted some indicator of progress. I could make it more explicit.